Documentation/Ctrl/Properties/PreviewRenderer.rst

```diff
@@ -0,0 +1,58 @@
+.. include:: /Includes.rst.txt
+.. _ctrl-reference-previewrenderer:
+
+===============
+previewRenderer
+===============
+
+.. confval:: previewRenderer
+
+   :Path: $GLOBALS['TCA'][$table]['ctrl']
+   :type: string
+   :Scope: Display
+
+   Configure a backend preview for a content element.
+
+Examples
+========
+
+Have also a look at :ref:`t3coreapi:ConfigureCE-Preview` for more details.
+
+Use for any record in a table
+-----------------------------
+
+This specifies the preview renderer to be used for any record in :sql:`my_table`:
+
+..  code-block:: php
+
+    $GLOBALS['TCA']['my_table']['ctrl']['previewRenderer']
+        = \MyVendor\MyExtension\Preview\PreviewRenderer::class;
+
+
+Table has a "type" field/attribute
+----------------------------------
+
+This specifies the preview renderer only for records of type :php:`$type` as
+determined by the type field of your table.
+
+..  code-block:: php
+
+    $GLOBALS['TCA']['my_table']['types'][$type]['previewRenderer']
+        = \MyVendor\MyExtension\Preview\PreviewRenderer::class;
+
+Table has a "subtype_value_field" setting
+-----------------------------------------
+
+If your table and field have a :php:`subtype_value_field` TCA setting (like
```

Comment:
Can we link this to further explainations?

Reply:
Done